sdk/core/System.ClientModel/src/Internal/CancellationHelper.cs

```diff
@@ -6,42 +6,46 @@
 
 namespace System.ClientModel.Internal;
 
-internal class CancellationHelper
+// Copy of https://github.com/dotnet/runtime/blob/b6dda7b719eab464d417904a4f4501b42cc10cdb/src/libraries/System.Net.Http/src/System/Net/Http/CancellationHelper.cs#L10
+internal static class CancellationHelper
```

Comment:
Updating to more closely match the Azure.Core version of this file.

Reply:
Just for my own understanding, why aren't we using shared source for these files? I assume we are not planning to maintain both copies.

Reply:
Yeah, it's a good question.  The current shared source files are in an Azure namespace, and our goal in System.ClientModel is to remove all Azure branding.  I've filed https://github.com/Azure/azure-sdk-for-net/issues/41035 to track removing this duplication -- once ClientModel GA's, we can either move Core and existing client to use the unbranded shared type (this may take some coordination with the generator), look at having the Azure.Core type use implementations in ClientModel, or making some of the types public in ClientModel.  We didn't want solving this issue now to add time to ClientModel release timelines.